setup.cfg

```diff
@@ -51,5 +53,6 @@ exclude =
     docs/,
     build,
     dist
-max-line-length = 99
```

Comment:
is there an important reason to change this?
I prefer the explicitness. Black makes it 99, flake8 also. I think that works, lmk if you disagree.

Reply:
I removed it for PEP8 compliancy, to preserve the [Flake8 default `max-line-length = 79`](https://flake8.pycqa.org/en/latest/user/options.html#cmdoption-flake8-max-line-length) - I didn't realize your extending it to `99` was an intended permanent deviation. :) The Black line length is `88`, so I changed it to `79` as well as in `setup.cfg` to match the Flake8 default (I know that following the Black default is also an option, but I've read that [some developers are still not convinced by Black's choice to deviate from PEP](https://github.com/psf/black/issues/498), so I went with the safe option). FYI this is the same config as used for the API!

The addition of
```
extend-ignore = E501
extend-select = B950
```
to the Flake8 config (in the next lines) was adopted from [Bugbear's opinionated warnings] (https://github.com/PyCQA/flake8-bugbear#how-to-enable-opinionated-warnings), and basically results in the behaviour of:

Reply:
Hey, thanks! I'm aware of the defaults and (mostly) the arguments for them. There is nothing wrong with these defaults either - and neither is there a big problem with allowing longer lines. So this comes down to personal preference, and my preference is to allow longer lines - partly because in my experience the added room allows you to use more descriptive naming conventions and that aids readibility. 

That said, consistency is important. So whichever way we go, we should use across all python codebases (right now API + CLI/here). If you and Arman feel strongly about using the black defaults, then I think that'd be a good enough reason to trump my personal preference.

Reply:
I see. Given that the Flake8/PEP8 default is on the shorter side, how about a compromise of using the Black default = 88? Not an expert, but I think if we're going to use a linter/style formatter, I'm inclined to follow at least one of the tools' recommended line lengths. 😛 @rmanaem, any opinions here between going with a max line length of 79 (Flake8), 88 (Black), or 99?

Reply:
I'm OK with going with the black defaults for consistency sake. It's really not that important (but that has obviously never prevented anyone including me from having strong feelings about it :) ). We could talk about single or double quoted strings maybe :fire:  ?

Reply:
Haha, oh no. I meant the thing about single / double quotes as a joke. Didn't realize this might become relevant so soon. Yes yes, let's stick with whatever the black defaults are for that as well. That makes sense to me!